python/hail/dataset.py

```diff
@@ -1433,53 +1471,51 @@ def export_vcf(self, output, append_to_header=None, export_pp=False, parallel=Fa
 
         >>> hc.read('data/example.vds').export_vcf('data.example.vcf.bgz', export_pp=True, parallel=True)
 
-        Export a sites-only VCF with addition header lines from `myheader.txt`:
+        Export a sites-only VCF with additional header lines from *myheader.txt*:
 
         >>> (hc.read('data/example.vds')
         >>>    .filter_samples_all()
         >>>    .export_vcf('data/sites-only.vcf.bgz', append_to_header='data/myheader.txt')
 
         **Notes**
-        py:meth:`exportvcf` exports the VDS the VCF format as described in the [VCF 4.2 spec](https://samtools.github.io/hts-specs/VCFv4.2.pdf).
+        py:meth:`exportvcf` writes the VDS to disk in VCF format as described in the `VCF 4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
 
-        - Use the ``.vcf.bgz`` extension in the output file name for `blocked GZIP <http://www.htslib.org/doc/tabix.html>`_ compression. We strongly recommended this for exporting large VCFs.
-        - Set ``parallel=True`` to return a set of VCF files (one per partition) rather than serially concatenating these files.
-        - Set ``append_to_header=file.txt`` to append *file.txt* to the output VCF header.
-        - Set ``export_pp=True`` to export linear-scaled probabilities (Hail's pp field on genotype) as the VCF PP FORMAT field.
+        Use the ``.vcf.bgz`` extension in the output file name for `blocked GZIP <http://www.htslib.org/doc/tabix.html>`_ compression.
 
-        Importing a VCF to a Hail VDS and then exporting to a VCF will not result in exactly the same VCF.  The differences, however, are limited to the header and INFO field.  The output VCF header will contain:
+        .. note::
+
+            We strongly recommended compressed and parallel output when exporting large VCFs.
 
-        - FORMAT lines, with the format "GT:AD:DP:GQ:PL", or "GT:AD:DP:GQ:PP" if ``export_pp=True``
-        - FILTER lines, if present in the imported VCF
-        - INFO lines (see exported annotations below)
+        Following import-export workflow
 
-        The output VCF header will not contain these lines unless they are explicitly copied to a file and inserted using the ``append_to_header`` option:
+        >>> hc.import_vcf('data/example.vcf').export_vcf('data/example_out.vcf')
 
-        - contig lines
-        - lines added by external tools (bcftools, GATK, ...)
+        the *example_out.vcf* header will contain the FORMAT, FILTER, and INFO lines present in *example.vcf*.  However, it will *not* contain CONTIG lines or lines added by external tools (such as bcftools and GATK) unless they are explicitly inserted using the ``append_to_header`` option.
```

Comment:
I still think this section is incredibly confusing. Maybe use bullet points and reduce the number of examples.